<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyWidget</class>
 <widget class="QWidget" name="PropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>703</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="documentPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>282</width>
        <height>625</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Document</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="defaultSettingsGroup">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="rowWrapPolicy">
           <enum>QFormLayout::DontWrapRows</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="dpiLabel">
            <property name="text">
             <string>DPI</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="sheetLabel">
            <property name="text">
             <string>Sheet</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QGridLayout" name="sheetAxesGrid">
            <item row="0" column="1">
             <widget class="QLabel" name="axesDirectionSelectLabel">
              <property name="text">
               <string>Axes</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="originSelectLabel">
              <property name="text">
               <string>Origin</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="RadioButtonGrid" name="originSelectGrid" native="true"/>
            </item>
            <item row="1" column="1">
             <widget class="RadioButtonGrid" name="axesDirectionSelectGrid" native="true"/>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="ValueSetter" name="dpiSetter" native="true"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="defaultTextGroup">
         <property name="title">
          <string>Text</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="rowWrapPolicy">
           <enum>QFormLayout::DontWrapRows</enum>
          </property>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="boldCheck">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>B</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="italicCheck">
              <property name="font">
               <font>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>I</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="underlineCheck">
              <property name="font">
               <font>
                <italic>false</italic>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>U</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="strikeoutCheck">
              <property name="font">
               <font>
                <italic>false</italic>
                <underline>false</underline>
                <strikeout>true</strikeout>
               </font>
              </property>
              <property name="text">
               <string>S</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QFontComboBox" name="defaultFontComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentFont">
             <font>
              <family>Arial</family>
             </font>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QUndoView" name="undoView"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="controlsPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>282</width>
        <height>625</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Controls</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="colorModelLabel">
         <property name="text">
          <string>Color Model</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="colorModelSelect">
         <item>
          <property name="text">
           <string>RGB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CMYK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HSL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HSV</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="strokeGroup">
         <property name="title">
          <string>Stroke</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ColorSetter" name="strokeColor" native="true"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="strokeThicknessLabel">
            <property name="text">
             <string>Thickness</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ValueSetter" name="widget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="fillGroup">
         <property name="title">
          <string>Fill</string>
         </property>
         <layout class="QFormLayout" name="formLayout_5">
          <item row="1" column="0" colspan="2">
           <widget class="QGroupBox" name="gradientGroup">
            <property name="title">
             <string>Gradient</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QFormLayout" name="formLayout_6">
             <item row="0" column="0">
              <widget class="QLabel" name="fillColorLabel_2">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="ColorSetter" name="fillColor_2" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="fillColorLabel">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ColorSetter" name="fillColor" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="fontGroup">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Font</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="boldCheck_2">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>B</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="italicCheck_2">
              <property name="font">
               <font>
                <italic>true</italic>
               </font>
              </property>
              <property name="text">
               <string>I</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="underlineCheck_2">
              <property name="font">
               <font>
                <italic>false</italic>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>U</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="strikeoutCheck_2">
              <property name="font">
               <font>
                <italic>false</italic>
                <underline>false</underline>
                <strikeout>true</strikeout>
               </font>
              </property>
              <property name="text">
               <string>S</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFontComboBox" name="defaultFontComboBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentFont">
             <font>
              <family>Arial</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValueSetter</class>
   <extends>QWidget</extends>
   <header>Windows/MainWindow/SubWidgets/valuesetter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorSetter</class>
   <extends>QWidget</extends>
   <header>Windows/MainWindow/SubWidgets/colorsetter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RadioButtonGrid</class>
   <extends>QWidget</extends>
   <header>Windows/MainWindow/SubWidgets/radiobuttongrid.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
